packages/client/src/containers/CreateTeamPage/CreateTeam.js

```diff
@@ -0,0 +1,177 @@
+import React, { useState } from 'react';
+import './CreateTeam.css';
+
+function CreateTeam() {
+  const [teamName, setTeamName] = useState('');
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [email, setEmail] = useState('');
+  const [teamMembers, setTeamMembers] = useState([]);
+  const employees = [
+    { id: 1, firstName: 'John', lastName: 'Doe', email: 'john@example.com' },
+    { id: 2, firstName: 'Jane', lastName: 'Smith', email: 'jane@example.com' },
+    {
+      id: 3,
+      firstName: 'Alice',
+      lastName: 'Johnson',
+      email: 'alice@example.com',
+    },
+  ];
+
+  const handleInputChange = (inputType, e) => {
+    if (inputType === 'firstNameInput') {
+      setFirstName(e.target.value);
+    } else if (inputType === 'lastNameInput') {
+      setLastName(e.target.value);
+    } else if (inputType === 'emailInput') {
+      setEmail(e.target.value);
+    }
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    if (!teamMembers.length) {
```

Comment:
Instead of doing this you could maybe disable the button 🤔

Reply:
Done